app/views/galleries/_icon_item.haml

```diff
@@ -1,3 +1,5 @@
+-# locals: ( icon_item:, is_owner: false, skip_forms: false )
```

Comment:
Does it make sense to have defaults in all these fields? I think these are obligatory in the parent

Reply:
They have defaults in `_icon_view`